Doc/library/test.rst

```diff
@@ -1691,6 +1691,21 @@ The :mod:`test.support.warnings_helper` module provides support for warnings tes
 .. versionadded:: 3.10
 
 
+.. function:: ignore_warnings(*, category)
+
+   A decorator to suppress warnings. *category* is a subclass of
+   :exc:`Warning` class. It is approximately equivalent to calling
+   :func:`warnings.catch_warnings()` with :meth:`warnings.simplefilter`
+   set to ``ignore``. If you want to suppress some specific known warnings
+   in a test case, use code like the following::
```

Comment:
Sorry to pick on this further, but I think this may be too verbose for the `test.support` docs, as they are primarily written for CPython core devs and contributors.

I'd prefer a simpler variant, kind of like this:

```suggestion
   :exc:`Warning`.
   Example use::
```

Alternative:

```suggestion
   the :exc:`Warning` class.
   Example use::
```

@CAM-Gerlach, what do you think?

Reply:
It could definitely say the same thing in a lot fewer words, though considering this whole page is specifically for internal use, I wouldn't cut out _that_ many relevant details.—in particular, the  Also, looking at it more, I spotted some other things that can be fixed/improved. I've made another suggestion covering the full block that does both (and uses SemBr).

Reply:
Looks like the agreed suggestion was applied, so marking this as resolved, thanks.